src/observability/summarizer.go

```diff
@@ -127,7 +127,7 @@ func convertSysLogToSysSummaryMap(syslogs []*pb.Alert) {
 		sysSummary.Deployment = ""
 
 		for _, d := range deployments {
-			if d.Labels == syslog.Labels && d.Namespace == syslog.NamespaceName {
+			if strings.Contains(syslog.Labels, d.Labels) && d.Namespace == syslog.NamespaceName {
```

Comment:
I need clarification in this logic @vishnusomank 
Why are we using strings.Contains instead of == ?

Reply:
@seswarrajan  The `syslog.Labels` sometimes contains multiple labels as a single string and when we use == to compare it with `d.Labels` it fails. So to reduce the failure rate I added the `strings.Contains` condition.

eg: 
syslog.Labels 
 ```
sidecar.istio.io/inject=false,app=istiod,install.operator.istio.io/owning-resource=unknown,istio=pilot istio.io/rev=default,operator.istio.io/component=Pilot
```
d.Labels
```
app=istiod
```

In this scenario, the pod belongs to the deployment `istiod` but if we use == it'll not be able to register.